packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -71,6 +68,13 @@ class _DeepLinkListViewState extends State<DeepLinkListView>
       ),
     );
   }
+
+  int _getReleaseVariantIndex(List<String> variants) {
+    final index = variants
+        .indexWhere((variant) => variant.toLowerCase().contains('release'));
```

Comment:
use the `caseInsensitiveContains` utility method from `shared/primitives/utils.dart`